published_versions.json

```
@@ -189,13 +226,256 @@
+          "cuda11.6": {
+            "note": "# MacOS Binaries dont support CUDA, install from source if CUDA is needed",
+            "command": "pip3 install torch torchvision torchaudio"
+          },
+          "rocm5.x": {
+            "note": "<b>NOTE:</b> ROCm is not available on MacOS",
+            "command": null
+          },
+          "accnone": {
+            "note": null,
+            "command": "pip3 install torch torchvision torchaudio"
+          }
+        },
+        "conda": {
+          "cuda10.2": {
+            "note": "# MacOS Binaries dont support CUDA, install from source if CUDA is needed",
+            "command": "conda install pytorch torchvision -c pytorch-test"
+          },
+          "cuda11.x": {
+            "note": "# MacOS Binaries dont support CUDA, install from source if CUDA is needed",
+            "command": "conda install pytorch torchvision -c pytorch-test"
+          },
+          "cuda11.6": {
+            "note": "# MacOS Binaries dont support CUDA, install from source if CUDA is needed",
+            "command": "conda install pytorch torchvision -c pytorch-test"
+          },
+          "rocm5.x": {
+            "note": "<b>NOTE:</b> ROCm is not available on MacOS",
+            "command": null
+          },
+          "accnone": {
+            "note": "# MacOS Conda binaries are for x86_64 only, for M1 please use wheels",
+            "command": "conda install pytorch torchvision -c pytorch-test"
+          }
+        },
+        "libtorch": {
+          "accnone": {
+            "note": null,
+            "versions": {
+              "Download here:": "https://download.pytorch.org/libtorch/test/cpu/libtorch-macos-1.12.0.zip"
+            }
+          },
+          "cuda10.2": {
+            "note": null,
+            "versions": {
+              "MacOS binaries do not support CUDA. Download CPU libtorch here:": "https://download.pytorch.org/libtorch/test/cpu/libtorch-macos-1.12.0.zip"
+            }
+          },
+          "cuda11.x": {
+            "note": null,
+            "versions": {
+              "MacOS binaries do not support CUDA. Download CPU libtorch here:": "https://download.pytorch.org/libtorch/test/cpu/libtorch-macos-1.12.0.zip"
+            }
+          },
+          "cuda11.6": {
+            "note": null,
+            "versions": {
+              "MacOS binaries do not support CUDA. Download CPU libtorch here:": "https://download.pytorch.org/libtorch/test/cpu/libtorch-macos-1.12.0.zip"
+            }
+          },
+          "rocm5.x": {
+            "note": "<b>NOTE:</b> ROCm is not available on MacOS",
+            "versions": null
+          }
+        }
+      },
+      "windows": {
+        "pip": {
+          "accnone": {
+            "note": null,
+            "command": "pip3 install torch torchvision torchaudio"
+          },
+          "cuda10.2": {
+            "note": "CUDA-10.2 PyTorch builds are no longer available for Windows, please use CUDA-11.3",
```

Is there a reason for suggesting 11.3 over 11.6?
```suggestion
            "note": "CUDA-10.2 PyTorch builds are no longer available for Windows, please use CUDA-11.6",
```